libcob/common.c

```diff
@@ -2502,6 +2511,12 @@ cob_set_exception (const int id)
 	}
 }
 
+int
+cob_last_exception_fatal() {
```

Comment:
`cob_last_exception_was_fatal (void)`, please - but that isn't used and needed any more, is it?